client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx

```diff
@@ -14,12 +14,7 @@ interface Props {
 export default function SitePreviewPane( { selectedSite, closeSitePreviewPane }: Props ) {
 	return (
 		<div className="site-preview__pane">
-			<SitePreviewPaneHeader
-				title={ selectedSite.blogname }
-				url={ selectedSite.url }
-				urlWithScheme={ selectedSite.url_with_scheme }
-				closeSitePreviewPane={ closeSitePreviewPane }
-			/>
+			<SitePreviewPaneHeader site={ selectedSite } closeSitePreviewPane={ closeSitePreviewPane } />
```

Comment:
The intention here of not using Site object, is to make the Preview Pane not couple with Site (follow-up PRs, to refactor it). So the header only needs those 4 fields: `title`, `url`, `urlWithScheme`, and `closeSitePreviewPane`.

Reply:
Everything here is site-centric, so any reason we wouldn't want to just pass the Site object? This prevents us from needing to pass tons of props multiple layers, and it also means it's more future-proof (add a key to the Site object (e.g. color), and it's automatically available).